src/ts/components/controlbar.ts

```diff
@@ -58,13 +61,27 @@ export class ControlBar extends Container<ControlBarConfig> {
       });
     }
 
+    if (BrowserUtils.isMobile) {
+      uimanager.onComponentShow.subscribe((component: Component<ComponentConfig>) => {
+        if (component instanceof SettingsPanel) {
+          isSettingsPanelShown = true;
+        }
+      });
+
+      uimanager.onComponentHide.subscribe((component: Component<ComponentConfig>) => {
+        if (component instanceof SettingsPanel) {
+          isSettingsPanelShown = false;
+        }
+      });
+    }
+
     uimanager.onControlsShow.subscribe(() => {
       this.show();
     });
 
     uimanager.onPreviewControlsHide.subscribe((sender, args) => {
-      // Cancel the hide event if hovered child components block hiding
-      args.cancel = (hoverStackCount > 0);
+      // Cancel the hide event if hovered child components block hiding or if the settings panel is active on mobile.
+      args.cancel = (hoverStackCount > 0 || isSettingsPanelShown);
```

Comment:
mabye do

` args.cancel = args.cancel || (hoverStackCount > 0 || isSettingsPanelShown);`

otherwise components subscribed to this earlier can't cancel.

Reply:
Agree, thanks for the suggestion.